packages/cosmic-swingset/test/make.test.ts

```diff
@@ -111,11 +104,16 @@ test.serial('integration test: rosetta CI', async t => {
   t.is(code, 0, 'make scenario2-run-rosetta-ci is successful');
 });
 
-/** @type {import('ava').Macro<[title: string, verifier?: any], TestContext>} */
-const walletProvisioning = test.macro({
-  title(_, title, _verifier) {
-    return title;
-  },
+interface Verifier {
+  start: (t: ExecutionContext<TestContext>) => Promise<void>;
+  stop: (t: ExecutionContext<TestContext>) => Promise<void>;
+}
+
+const walletProvisioning: Macro<
+  [title: string, verifier?: Verifier],
+  TestContext
+> = test.macro({
+  title: (_provided, title, _verifier) => title,
   async exec(t, _title, verifier) {
```

Comment:
Better still would be dropping the trivial `title` method entirely, cf. [Reusing test logic through macros](https://github.com/avajs/ava/blob/main/docs/01-writing-tests.md#reusing-test-logic-through-macros).
```ts
const walletProvisioning: Macro<[verifier?: Verifier], TestContext> =
  test.macro(async (t, verifier) => {
    …
  });
…
test.serial('wallet provisioning', walletProvisioning);
…
test.serial('vstorage metrics', walletProvisioning, {
  start: async t => {…},
  stop: async t => {…},
});
```